docs/training/llm_trace_usage.zh.md

```
@@ -0,0 +1,180 @@
+
+### 2. 配置文件
+
+你也可以在 `agent.yml` 配置文件中为特定模型启用跟踪：
+
+```yaml
+agent:
+  models:
+    deepseek-v3:
+      type: deepseek
+      base_url: https://api.deepseek.com
+      api_key: ${DEEPSEEK_API_KEY}
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)